.github/workflows/ci-wheels.yml

```
@@ -78,14 +98,24 @@ jobs:
         CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
         CIBW_BEFORE_BUILD_LINUX: yum install -y udunits2-devel
         CIBW_BEFORE_BUILD_MACOS: brew install udunits
+        CIBW_BEFORE_BUILD_WINDOWS: python -m pip install delvewheel
         CIBW_TEST_REQUIRES: pytest
         CIBW_TEST_COMMAND: >
           python -c 'import cf_units; print(f"cf-units v{cf_units.__version__}")' &&
           python -m pytest --pyargs cf_units
+        CIBW_TEST_COMMAND_WINDOWS: >
+          python -c 'import cf_units; print(f"cf-units v{cf_units.__version__}")' ;
```

Apologies - this is still failing.
I think it is down to the quotes used (windows seems to be more picky than linux)
Hopefully this will fix it. Fingers crossed.

```suggestion
          python -c "import cf_units; print(f'cf-units v{cf_units.__version__}')" ;
```